feat(manager): Adds Unity3D (#28189)
renovatebot/renovate: lib/modules/manager/unity3d/readme.md

> Handles version numbers inside `ProjectSettings/ProjectVersion.txt` files used by the Unity game engine.

Reviewer: I see two ways to read this:

- Renovate can read `ProjectSettings.txt` and `ProjectVersion.txt` files
- Renovate can read a `ProjectVersion.txt` file _if_ its in the `ProjectSettings` directory

I don't know which interpretation is correct, as I don't use Unity.

Please update the text so it explains precisely what Renovate can update. 🙂

> `ProjectSettings/ProjectVersion.txt` always contains two version references in a yml-like syntax; one with and one without a hash, both of which are updated:

Reviewer: ```suggestion
`ProjectSettings/ProjectVersion.txt` always have two version references in a yml-like syntax; one with and one without a hash, both are updated:
```

Also, is it YML-like syntax or YAML-syntax? I lean towards YAML-syntax, but not sure. 🙂

```yml
m_EditorVersion: 2020.3.15f2
m_EditorVersionWithRevision: 2020.3.15f2 (6cf78cb77498)
```

> Relies on the corresponding `unity3d` datasource and versioning.

Reviewer: I don't see any changes to a datasource directory in this PR? Maybe that was added in a different PR? Or I'm misunderstanding something here. 🙃

In any case, please check if this sentence is correct.
